schemas/cellCultureType.schema.tpl.json

```diff
@@ -0,0 +1,4 @@
+{
+    "_type": "https://openminds.ebrains.eu/controlledTerms/CultureType",
```

Comment:
cellCultureType or you need to rename the file

Reply:
done